vipul/Gallery_System/portrait.html

```diff
@@ -0,0 +1,102 @@
+							<li><a href="portrait.html">Portrait</a></li>
+							<li><a href="fashion.html">Fashion</a></li>
+							<li><a href="macro.html">Macro</a></li>
+							<li><a href="bird.html">Bird</a></li>
+							<li><a href="nature.html">Nature</a></li>
+							<li><a href="food.html">Food</a></li>
+						</ul>
+					</li>
+					<li><a href="services.html">SERVICES</a></li>
+					<li><a href="about.html">ABOUT</a></li>
+					<li><a href="contact.html">CONTACT</a></li>
+				</ul>
+			</div>
+		</div>
+		<div class="category-wild">
+			<div class="gallery-wild">
+					<a target="_blank" href="images/portrait1.jpeg">
+						<img src="images/portrait1.jpeg" alt="Picture of Portrait Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/portrait2.jpg">
+						<img src="images/portrait2.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/portrait3.jpg">
+						<img src="images/portrait3.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/portrait4.jpg">
+						<img src="images/portrait4.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/portrait5.jpg">
+						<img src="images/portrait5.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/portrait6.jpg">
+						<img src="images/portrait6.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div>
+			<div class="gallery-wild">
+					<a target="_blank" href="images/portrait7.jpg">
+						<img src="images/portrait7.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div><div class="gallery-wild">
+					<a target="_blank" href="images/portrait8.jpg">
+						<img src="images/portrait8.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div><div class="gallery-wild">
+					<a target="_blank" href="images/portrait9.webp">
+						<img src="images/portrait9.webp" alt="Picture of Portrait Photography">
+					</a>
+			</div><div class="gallery-wild">
+					<a target="_blank" href="images/portrait10.jpg">
+						<img src="images/portrait10.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div><div class="gallery-wild">
+					<a target="_blank" href="images/portrait1.jpeg">
+						<img src="images/portrait1.jpeg" alt="Picture of Portrait Photography">
+					</a>
+			</div><div class="gallery-wild">
+					<a target="_blank" href="images/portrait2.jpg">
+						<img src="images/portrait2.jpg" alt="Picture of Portrait Photography">
+					</a>
+			</div>
+			
+		</div>
+		<div class="footer">
+			<p>
+				Copyright 2020 All rights reserved | This is developed by <a href="https://www.facebook.com/vipul.suryawanshi" target="_blank">Vipul Suryawanshi.</a>
+			</p>
+		</div>
+	</body>	
+</html>	
```

Comment:
remove all extra spaces
Add more content in footer
Add contact us in footer
bad code indentation

Reply:
i will do all the changes